src/improvements/tasks/eth/018-U16-opcm-upgrade-v400-soneium/VALIDATION.md

```diff
@@ -83,3 +83,328 @@ The resulting calldata sent from the ProxyAdminOwner safe is thus:
+  - **After:** [`0xeb69cC681E8D4a557b30DFFBAd85aFfD47a2CF2E`](https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-mainnet.toml#L13)
+  - **Summary:** ERC-1967 implementation upgraded to AnchorStateRegistry v3.5.0
+  - **Detail:** Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+  
+- **Key:**          `0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103`
+  - **Decoded Kind:** `address`
+  - **Before:** `0x0000000000000000000000000000000000000000`
+  - **After:** [`0x89889B569c3a505f3640ee1Bd0ac1D557f436D2a`](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/mainnet/soneium.toml#L60)
+  - **Summary:** Proxy owner set to ProxyAdminOwner for Soneium
+  - **Detail:** Standard slot for storing the owner address in a Proxy contract.
+  
+  ---
+  
+### `0x512a3d2c7a43bd9261d2b8e8c9c70d4bd4d503c0` ([DisputeGameFactoryProxy](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/mainnet/soneium.toml#L53)) - Chain ID: 1868
+  
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  - **Decoded Kind:** `address`
+  - **Before:** `0x4bbA758F006Ef09402eF31724203F316ab74e4a0`
+  - **After:** [`0x33D1e8571a85a538ed3D5A4d88f46C112383439D`](https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-mainnet.toml#L16)
+  - **Summary:** ERC-1967 implementation upgraded to DisputeGameFactory v1.2.0
+  - **Detail:** Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+  
+- **Key:**          `0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e`
+  - **Before:** [`0x0000000000000000000000003d56d47b9e7e34a46612badc70377f74051e6b17`](https://github.com/ethereum-optimism/superchain-registry/blob/6621a0f13ce523fe1bb8deea739fe37abe20f90d/superchain/configs/mainnet/soneium.toml#L67)
+  - **After:** `0x0000000000000000000000003c12f1f4f0702cb7fc83e2e5594331c10b9e39b4`
+  - **Summary:** gameImpls mapping updated - GameType 1 implementation changed
+  - **Detail:** Mapping slot derivation for gameImpls[GameType(1)] - updated game implementation address for GameType 1
+  
+  ---
+  
+### `0x56ebc5c4870f5367b836081610592241ad3e0734` ([OPContractsManager](https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-mainnet.toml#L22)) - Chain ID: 1
+  
+- **Key:**          `0x0000000000000000000000000000000000000000000000000000000000000001`
+  - **Before:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+  - **After:** `0x0000000000000000000000000000000000000000000000000000000000000000`
+  - **Summary:** _initializing flag reset to false (initialization process completed)
+  - **Detail:** Reinitializable pattern - marks initialization process as complete
```

Comment:
1. this is wrong, it's not the init value, it's the `isRC` flag.
2. it should no longer appear in the diff now that the override has been added to the config.toml

Reply:
```suggestion
```